backend/btrixcloud/models.py

```diff
@@ -1063,34 +1230,86 @@ class Collection(BaseMongoModel):
     name: str = Field(..., min_length=1)
     oid: UUID
     description: Optional[str] = None
+    caption: Optional[str] = None
     modified: Optional[datetime] = None
 
     crawlCount: Optional[int] = 0
     pageCount: Optional[int] = 0
     totalSize: Optional[int] = 0
 
+    dateEarliest: Optional[datetime] = None
+    dateLatest: Optional[datetime] = None
+
     # Sorted by count, descending
     tags: Optional[List[str]] = []
 
     access: CollAccessType = CollAccessType.PRIVATE
 
+    homeUrl: Optional[AnyHttpUrl] = None
+    homeUrlTs: Optional[datetime] = None
+    homeUrlPageId: Optional[UUID] = None
+
+    thumbnail: Optional[ImageFile] = None
+
 
 # ============================================================================
 class CollIn(BaseModel):
     """Collection Passed in By User"""
 
     name: str = Field(..., min_length=1)
     description: Optional[str] = None
+    caption: Optional[str] = None
     crawlIds: Optional[List[str]] = []
 
     access: CollAccessType = CollAccessType.PRIVATE
 
 
 # ============================================================================
-class CollOut(Collection):
+class CollOut(BaseMongoModel):
     """Collection output model with annotations."""
 
+    name: str
+    oid: UUID
+    description: Optional[str] = None
+    caption: Optional[str] = None
+    modified: Optional[datetime] = None
+
+    crawlCount: Optional[int] = 0
+    pageCount: Optional[int] = 0
+    totalSize: Optional[int] = 0
+
+    dateEarliest: Optional[datetime] = None
+    dateLatest: Optional[datetime] = None
+
+    # Sorted by count, descending
+    tags: Optional[List[str]] = []
+
+    access: CollAccessType = CollAccessType.PRIVATE
+
+    homeUrl: Optional[AnyHttpUrl] = None
+    homeUrlTs: Optional[datetime] = None
+    homeUrlPageId: Optional[UUID] = None
+
     resources: List[CrawlFileOut] = []
+    thumbnail: Optional[ImageFileOut] = None
+
+
+# ============================================================================
+class PublicCollOut(BaseMongoModel):
+    """Collection output model with annotations."""
+
+    name: str
```

Comment:
We'll need `oid` in the public response to construct the replay link

Reply:
Added!